src/server/components/dashboard/helpers.ts

```diff
@@ -21,21 +21,16 @@ export async function getInitiateFormRunnerSessionToken(
   const token = await axios
     .post(`http://${formRunnerNewSessionUrl}`, formRunnerWebhookData)
     .then((response) => {
-      logger.info(`response received from formRunnerNewSessionUrl: data ${response?.data}`);
-      Object.entries(response).map(([key, value]) =>
-        logger.info(`formRunnerNewSessionUrl response ${key} + ":" ${value}`)
-      );
-      Object.entries(response?.data).map(([key, value]) =>
-        logger.info(`formRunnerNewSessionUrl response.data ${key} + ":" ${value}`)
-      );
       return response?.data?.token;
     })
     .catch((error) => {
-      logger.error(`Error received after calling formRunnerNewSessionUrl ${error}`);
+      logger.error(
+        `getInitiateFormRunnerSessionToken: Error received after calling form runner URL ${formRunnerNewSessionUrl}. ${error}`
+      );
       throw new Error("Unable to initiate form runner session");
     });
 
-  logger.info(`token: ${token}`);
+  logger.info(`getInitiateFormRunnerSessionToken: formRunner responded with ${token}`);
```

Comment:
could you add logs in `initialiseFormRunnerSession` as well please? to log the listItem.id and token together, so it makes it easier to track which token belongs to the user. They're hard to track when lots of annual reviews are coming in - or different change requests are coming through at the same time

Reply:
Is the `listItem.id` accessible in the `getInitiateFormRunnerSessionToken` function?